### ISSUE TITLE
Continuously terminates and restarts

### ISSUE THREAD

**Issue by SinisterCrayon** [DyonR/docker-owncloud-client#2]
Container is set up and syncs normally. However, when it reaches the end of the sync it terminates and restarts the container instead of waiting to re-run owncloud client. This means that my array is in a constant state of "chown"ing the files, then running the sync, then "chown"ing the files again causing my array to fail to idle.

I changed uid to match the uid of the folder owner, and tried setting the sync interval to up to an hour but it seems to ignore this setting entirely.

**Comment by DyonR** [comment]
I haven't been working actively on my Docker containers, and admittedly completely missed that you opened issue, which I am really sorry for.  
However, I would still love to troubleshoot this. If you are still using this Docker, please let me know so we can look into this issue.

**Comment by SinisterCrayon** [comment]
Hey there! Thanks for checking in with me.

I am indeed still using the Docker container but I resolved the issue. It appears that when the sync fails for any reason (corrupted or missing file for example) the container restarts itself entirely rather than dealing with the error cleanly. I monitored logs to find this (which was a pain), but once I figured it out I just ran through the files that were showing as corrupt and deleted them from the Nextcloud server I was syncing from.

There were only a handful, and it looked like they had been casualties of a move from Owncloud to Nextcloud. Now it's working fine. :)
